src/com/team4/employeemood/User.java

```diff
@@ -17,6 +25,24 @@ public User(String firstName, String lastName, String birthDate, String employme
 
     public User(){
         //TO DO
```

Comment:
Nu cred ca are rost sa avem acest cod de validare regex in constructor, in validator si in objectcreator